### PR TITLE
Add missing page links to sitenav and workingWithSites overview

### LINKED ISSUE #2219: Add the "templates" page into the sitenav for more visibility
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### Any related issues?

_No response_

### What is the area that this feature belongs to?

Documentation

### Is your feature request related to a problem? Please describe.

Somehow this [page](https://markbind.org/userGuide/templates.html) is not included in the sitenav, making it less visible to new users.

### Describe the solution you'd like

Include it, perhaps under "Working with sites"

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by EssWhyy**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Fix #2219 (Add the "templates" page into the sitenav for more visibility)

**Anything you'd like to highlight/discuss:**
I have added the "templates" page to the sitenav and "Working With Sites" overview. I have also added the "Redirecting to 404" page section to the overview for consistency.

**Testing instructions:**

NA

**Proposed commit message: (wrap lines at 72 characters)**

Add missing page links to sitenav and workingWithSites overview

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [ ] No unrelated changes

### CODE REVIEW

**Comment by tlylt** [comment]
> I have also added the "Redirecting to 404" page section to the overview for consistency.

Do you mean "Adding Navigation Buttons" page?

**Review by tlylt — CHANGES_REQUESTED** [review]
Thank you for updating the docs! To complete the inclusion of both Templates and Adding Navigation Buttons, will need you to also update their respective navigation button at the bottom of both pages.

E.g. this place:
image
image
Read this https://markbind-master.netlify.app/userguide/addingnavigationbuttons and see other examples for details.

**Comment by EssWhyy** [comment]
Oh yep my mistake, yep I was referring to the Adding Navigation Buttons section, and yeah ok I will add in the navigation buttons for both pages

File: docs/userGuide/templates.md [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-# Templates
+<span id="link" class="d-none">
+<md>[_User Guide → {{ title }}_]({{ filename }}.html)</md>
+</span>
+
+# {{ title }}
 
-MarkBind allows you to start off your new project with different templates.
+<div class="lead" id="overview">
+  MarkBind allows you to start off your new project with different templates.
```
Comment (tlylt) [inline docs/userGuide/templates.md:18]:
```suggestion
\
MarkBind allows you to start off your new project with different templates.
```

**Comment by tlylt** [comment]
@EssWhyy btw for the PR description, note that to have GitHub detect and close an issue for you, you will need to use one of the closing keywords: "Fixes #xxx" or "Resolves #xxx" (at least for now, until they make it more intelligent with their new AI features). `Fix issue #2219` doesn't work (will need to remove the word "issue"), so `Fix #2219` will work.

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Before:
image
After I edited the description:
image

**Review by tlylt — COMMENTED** [review]
LGTM

**Review by tlylt — APPROVED** [review]

**Comment by tlylt** [comment]
@all-contributors please add @EssWhyy for doc

**Pull request merged**